src/modules/sheba/index.ts

```diff
@@ -60,6 +60,8 @@ export function shebaIso7064Mod97(iban: string): number {
  * @since 1.7.1
  */
 export function isShebaValid(shebaCode: string): boolean {
```

Comment:
it's better to convert shebaCode varible to upper case and then check for IR string

```ts
shebaCode = shebaCode.toUpperCase();
```

Reply:
Thanks for this great recommendation